README.md

```
@@ -1,6 +1,706 @@
+    WSClient->>API: Connect + Auth Token
+    API->>WSClient: Connection Established
+    API->>WSClient: Data Update Event
+    WSClient->>LiveProvider: Subscription Data
+    LiveProvider->>Component: Trigger Refetch
+```
+
+## Code Structure
+
+### Directory Organization
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)